### DISCUSSION TITLE
Ajuda para obter dados para o  TCC

### DISCUSSION THREAD

**Discussion by dinnymarks** [frontendbr/forum#2424] [Perguntas e Respostas]
Olá, pessoal! Estou precisando de ajuda com dados para meu TCC, é sobre a importância da língua inglesa na área da programação. Eu super agradeço a quem puder tirar um tempinho para responder ao questionário para me auxiliar. Desde já agradeço a todos!([https://forms.gle/9nwQKDPnwMFwxxyeA])

**Comment by Diego-Girao** [comment]
Done ✅
